### ISSUE TITLE
Scrolling when zoomed may have momentum

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molecules-and-light#343]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/QA/issues/503.
When zoomed in and moving the screen around, there can be momentum to the scroll, meaning that it keeps moving for a bit even after you have stopped actively moving it. This is seen when scrolling using the scroll wheel (spinning it or pressing down and moving it). This feels a bit odd to me, but if it is intended behavior feel free to close.
**Steps to reproduce**
1. Zoom in on sim
2. Press down scroll wheel to bring up 4-direction scroll
3. Move mouse to start scroll
4. Lift up on wheel to stop scroll (note drift after this)

**Visuals**
![scrollmomentum](https://user-images.githubusercontent.com/41024075/82599293-42a1e900-9b69-11ea-9c35-c50b3628bfac.gif)

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Molecules and Light‬
URL: https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-dev.50/phet/molecules-and-light_all_phet.html?zoom=true
Version: 1.5.0-dev.50 2020-05-12 15:11:56 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36
Language: en-US
Window: 1536x722
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe, this is incorrect behavior.

**Comment by jessegreenberg** [comment]
@KatieWoe can you please verify that this is fixed in master? query param for phettest is ?zoom=true

**Comment by KatieWoe** [comment]
Looks good on Win 10 Chrome master in regards to clicking down on the scroll wheel. I did notice that some of the "floating" behavior seem to happen when you use the scroll wheel too. It's a bit hard to tell if that is an issue though.

**Comment by jessegreenberg** [comment]
Thanks @KatieWoe sounds good - floating with the wheel is intentional, as it looks good with trackpad input and matches Mac behavior with a mouse. If it is annoying or problematic we can change it, but it feels OK to me.

I am going to close this issue as the main problem is fixed.
